website/client/src/components/achievements/levelUp.vue

```diff
@@ -1,109 +1,197 @@
 <template>
   <b-modal
     id="level-up"
-    :title="$t('levelUpShare')"
     size="sm"
-    :hide-footer="true"
-    :hide-header="true"
+    :title="title"
+    ok-only
+    :ok-title="$t('onwards')"
+    v-bind:footer-class="{ greyed: displayRewardQuest }"
   >
-    <div class="modal-body text-center">
-      <h2>{{ $t('reachedLevel', {level: user.stats.lvl}) }}</h2>
+    <section class="flex">
+      <span
+        class="star-group mirror"
+        v-html="starGroup"
+      ></span>
       <avatar
         class="avatar"
         :member="user"
       />
-      <p class="text">
-        {{ $t('levelup') }}
-      </p>
-      <button
-        class="btn btn-primary"
-        @click="close()"
-      >
-        {{ $t('onwards') }}
-      </button>
-      <br>
-      <!-- @TODO: Keep this? .checkboxinput(type='checkbox', v-model=
+      <span
+        class="star-group"
+        v-html="starGroup"
+      ></span>
+    </section>
+
+    <hr>
+    <p class="text">
```

Comment:
v-once

Reply:
Can't I theoretically just put v-once on the entire modal?

Reply:
nvm, that doesn't work, because then the title doesn't update between levels.

Reply:
Done.